General/Glossary.md

```diff
@@ -53,6 +53,9 @@ It is also used by other programming languages such as [Squeak](#squeak), Newspe
 
 ## Context
 
+A *Context* represent a program execution and store information about it. It will store for example the `CompiledMethod` currently been executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...
```

Comment:
```suggestion
A *Context* represent a program execution. It will store for example the `CompiledMethod` currently been executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...
```